ext/intl/listformatter/listformatter_class.c

```diff
@@ -135,17 +135,17 @@ PHP_METHOD(IntlListFormatter, format)
     zval *val;
 
     ZEND_HASH_FOREACH_VAL(ht, val) {
-        zend_string *str_val;
+        zend_string *str_val, *tmp_str;
         
-        str_val = zval_get_string(val);
+        str_val = zval_get_tmp_string(val, &tmp_str);
```

Comment:
Isn't there an exception check missing here if we have a non stringable object (or an error handler promoting the array to string warning) in `val`?

Reply:
I guess, I'm just used with us dealing with those sorts of things more proactively now, and I guess this _can_ matter if there is a follow-up object that does some stuff with side-effects in ``__toString()`` ?

Reply:
Ah I forgot that the VM doesn't actual call follow-up methods in case of an Exception.